lib/util/fs/index.ts

```diff
@@ -162,3 +163,30 @@ export function localPathIsFile(pathName: string): Promise<boolean> {
     .then((s) => s.isFile())
     .catch(() => false);
 }
+
+/**
+ * Find a file or directory by walking up parent directories within localDir
+ */
+
+export async function findUpLocal(
+  fileName: string | string[],
+  cwd: string
+): Promise<string | undefined> {
+  const { localDir } = GlobalConfig.get();
+  const absoluteCwd = upath.join(localDir, cwd);
+  const res = await findUp(fileName, { cwd: absoluteCwd, type: 'file' });
+  // Return undefined if nothing found
+  if (res === undefined) {
+    return res;
+  }
+  // Return relative path if file is inside of local dir
+  if (res.startsWith(localDir)) {
```

Comment:
This probably fails on windows.

Reply:
I tried to normalize the path, since it is how it worked before.